profile/README.md

```diff
@@ -0,0 +1,11 @@
+## Hi there 👋
+
+The organization aims to explain modern and complex ML tech to the community. Understanding any ML component comprehensively involves of three things:
```

Comment:
Org is a powerful word. I will avoid using it as much as possible. Maybe something like this?

We aim to explain new and complex ML tech to the community. This is our way of giving back to the awesome ML community. Understanding....